widget/embedded/src/components/ConfirmWalletsModal/WalletList.tsx

```diff
@@ -42,6 +41,7 @@ import {
   WalletImageContainer,
 } from './WalletList.styles';
 
+const EllipsisLength = 7;
```

Comment:
Can you ensure that the constants follow our naming convention? Additionally, I believe there might be an error in the name.
ACCOUNT_ADDRESS_MAX_CHARACTERS

Reply:
fixed